vector/src/main/java/im/vector/app/core/platform/VectorBaseActivity.kt

```diff
@@ -324,7 +334,7 @@ abstract class VectorBaseActivity<VB : ViewBinding> : AppCompatActivity(), Maver
     override fun onResume() {
         super.onResume()
         Timber.i("onResume Activity ${javaClass.simpleName}")
-
+        screenEvent = analyticsScreenName?.let { ScreenEvent(it) }
```

Comment:
We track on exit because we want to report the duration, and this is managed by `ScreenEvent`. It's a bit weird, I know, generally the duration could be managed by the analytics tools (since a screen replaces another, duration can be deduced). But as we do not track every screens it probably better like that

Reply:
makes sense, thanks for explaining!